containers/Dockerfile

```diff
@@ -37,9 +37,10 @@ COPY moz_crlite_lib /app/moz_crlite_lib
 COPY create_filter_cascade /app/create_filter_cascade
 COPY moz_kinto_publisher /app/moz_kinto_publisher
 COPY workflow /app/workflow
+COPY containers/scripts /app/scripts
 COPY setup.py /app/
 RUN pip3 install /app/moz_crlite_lib/ /app/
-
+RUN pip3 install --user moz_crlite_query
```

Comment:
We may need to move this line after the `USER` directive.

Reply:
I think it's better if it not be modifiable by the user account, so removing `--user` and replacing the relevant path in the calling script with `/usr/local/bin`

Reply:
(That's in b94c10a738707f42af405354172e645a757a813c)